pkg/GAPJulia/JuliaInterface/read.g

```diff
@@ -6,23 +6,22 @@
 ReadPackage( "JuliaInterface", "gap/JuliaInterface.gi");
 ReadPackage( "JuliaInterface", "gap/BindCFunction.gi" );
 
-dirs:= DirectoriesPackageLibrary( "JuliaInterface", "julia" );
-
-JuliaIncludeFile( Filename( dirs, "gaptypes.jl" ) );
-
-dirs:= DirectoriesPackageLibrary( "JuliaInterface", "../LibGAP.jl/src" );
-
 if not IsBound( JULIAINTERNAL_LOADED_FROM_JULIA ) then
-    JuliaEvalString( Concatenation( "Base.include( Main,\"", Filename( dirs, "initialization.jl" ), "\")" ) );
+    JuliaEvalString( "__IS_LOADED_FROM_GAP = true" );
```

Comment:
Perhaps rename `__IS_LOADED_FROM_GAP` to say `__JULIAINTERNAL_LOADED_FROM_GAP` to match `JULIAINTERNAL_LOADED_FROM_JULIA` (and then perhaps also rename the latter to `__JULIAINTERNAL_LOADED_FROM_JULIA` ?

Also, while in GAP it is perfectly normal to pollute the global namespace with variables (sadly, and due to the lack of proper namespaces), I suspect this is less well received on the Julia side. So I wonder if we can pull this off by some other means? E.g. could we set this value inside the `__JULIAGAPMODULE`? (But this is very important, and can easily be done (or not) after this PR is merged.

Reply:
Nope, this has to stay in `Main`, because it is older than the creation of the `GAP` module, as we read `GAP.jl` only afterwards.

`I will rename it to `__GAPINTERNAL_LOADED_FROM_GAP`, as opposed to `__JULIAINTERNAL_LOADED_FROM_JULIA`, to also emphasize in which runtime those vars exist.

Reply:
As I think it is weird to have a `__JULIAINTERNAL` variable in julia itself.